FLEDGE.md

```diff
@@ -121,6 +122,23 @@ The `adComponents` field contains the various ad components (or "products") that
 The browser will provide protection against microtargeting, by only rendering an ad if the same rendering URL is being shown to a sufficiently large number of people (e.g. at least 100 people would have seen the ad, if it were allowed to show).  As discussed in the [Outcome-Based TURTLEDOVE](https://github.com/WICG/turtledove/blob/master/OUTCOME_BASED.md) proposal, this threshold applies only to the rendered creative; all of the metadata associated with ad bidding is not under any such restriction.  Since a single interest group can carry multiple possible ads that it might show, the group will have an opportunity to re-bid another one of its ads to act as a "fallback ad" any time its most-preferred choice is below threshold.  This means that a small, specialized interest group that is still below the daily-update threshold could still choose to participate in auctions, bidding with a more-generic ad until the group becomes large enough.
 
 
+#### 1.3 Permission Delegation
+
+When one domain calls joinAdInterestGroup() or leaveAdInterestGroup() for an interest group with a different owner, the browser will fetch the URL https://owner.domain/.well-known/interest-group/permissions/domain.of.frame.that.called.the.method, without credentials, using the Network Partition Key of the frame that invoked the method. The fetched response should have a JSON MIME type and be of the format:
+
+```
+{ "joinAdInterestGroup": true/false,
+  "leaveAdInterestGroup": true/false
+}
+```
+
+Indicating whether the origin in the path has permissions to join and/or leave interest groups owned by the domain the request is sent to. Missing permissions are assumed to be false.
+
+The browser may limit, per page or globally, the number of interest groups joins waiting for .well-known fetches at a time, and drop additions that exceed this limit. It is recommended that at least 20 be allowed per live page at a time.
```

Comment:
> 1. nit: WDYT about changing "waiting" to "concurrently waiting" or "queued"?  I ask just as a clarification to emphasize the goal.

Done.

> 2. can we add clarification of the motive here?  I assume it's to prevent overwhelming the browser's networking stack? or consuming too much memory?

It's to prevent too much memory being used in the browser process at once, and to allow adding a user to IGs on navigate away.

I'm reluctant to clarify here until we decide what direction we want to go with.

We could alternatively just cancel pending joins on navigate away and cache them in the renderer process (this does give a compromised renderer the exact same timing information as promises would provide, though I don't think that's a huge deal).

Or we could use some other scheme.